src/customTheme.ts

```diff
@@ -39,11 +45,80 @@ const blueColor: ColorHues = {
   900: '#00131b',
 }
 
+// ***** New colorscheme *****
+
+// The coolGray palette contains all the shades of gray we'll use in the app.
+// Example uses: body text, headings, changelog text, page background.
+// https://tailwindcss.com/docs/customizing-colors#color-palette-reference
+
+const tailwind_coolGray: ColorHues = {
```

Comment:
[ESLint `camelcase` rule](https://eslint.org/docs/rules/camelcase#require-camelcase-camelcase) is not enabled but we should have all the constants purely in camelCase 😅 

FYI: the rule is not enabled since we receive keys in snake_case from GitHub API and the rule would complain there. Something I could solve by mapping the keys received, but I didn't have time to do that.

Reply:
So from `tailwind_coolGray` (snakecase) ->`tailwindCSSCoolGray` (camelcase) then?

Reply:
That's right, or even `tailwindCoolGray` for simplicity!

Reply:
> tailwindCoolGray

That's a good name!